HWs/Test1.1/task2.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+
+void printArr(int array[], int lengthArray)
```

Comment:
```suggestion
void printArray(int array[], int lengthArray)
```
Не нужно сокращать название без веской причины.